research/deep_speech/data/download.py

```diff
@@ -0,0 +1,171 @@
+#  Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""Download and preprocess LibriSpeech dataset for DeepSpeech model."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import codecs
+import fnmatch
+import os
+import tarfile
+import tempfile
+import unicodedata
+
+from absl import app as absl_app
+from absl import flags as absl_flags
+import pandas
+from six.moves import urllib
+from sox import Transformer
+import tensorflow as tf
+
+LIBRI_SPEECH_URLS = {
```

Comment:
Could you add more information about the dataset in comments (or maybe README later)? Like the dataset size, and how long it takes to download.

Reply:
Not sure where we put data specs in the past, but adding to README sounds good to me.